From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeremy Apthorp <jeremya@chromium.org>
Date: Wed, 8 Jan 2020 16:06:14 -0800
Subject: FIXME: grit conflicts

This works around a bug in grit's handling of generated .grd files.

Should be removed once grit is fixed.

Tracking bug: https://crbug.com/1040605

diff --git a/tools/gritsettings/resource_ids.spec b/tools/gritsettings/resource_ids.spec
index e1a88b4c3ab4ad3f502020765f076b0e0387a7ad..d7e2fe39b2897cb23ca6efd29d904cc7880d3c26 100644
--- a/tools/gritsettings/resource_ids.spec
+++ b/tools/gritsettings/resource_ids.spec
@@ -581,15 +581,16 @@
     "includes": [3840],
   },
 
+  # FIXME: put this back after devtools_resources once grit is fixed to respect META properly
+  "electron/electron_resources.grd": {
+    "messages": [3860],
+    "includes": [4100],
+  },
+
   # This file is generated during the build.
   "<(SHARED_INTERMEDIATE_DIR)/devtools/devtools_resources.grd": {
     "META": {"sizes": {"includes": [500],}},
-    "includes": [3860],
-  },
-
-  "electron/electron_resources.grd": {
-    "messages": [31750],
-    "includes": [31950],
+    "includes": [4860],
   },
 
   # END "everything else" section.
